test/powershell/engine/Formatting/ErrorView.Tests.ps1

```diff
@@ -34,12 +41,11 @@ Describe 'Tests for $ErrorView' -Tag CI {
                 $b = 2
 '@
 
-            $testScriptPath = Join-Path -Path $TestDrive -ChildPath 'test.ps1'
             Set-Content -Path $testScriptPath -Value $testScript
-            $e = { & $testScriptPath } | Should -Throw -ErrorId 'UnexpectedToken' -PassThru
-            $e | Out-String | Should -BeLike "*${testScriptPath}:4*"
+            $e = { & $testScriptPath } | Should -Throw -ErrorId 'UnexpectedToken' -PassThru | Out-String
+            $e | Should -BeLike "*${testScriptPath}:4*"
             # validate line number is shown
-            $e | Out-String | Should -BeLike '* 4 *'
+            $e | Should -BeLike '* 4 *'
```

Comment:
I'm not sure why there are 2 assertions here, it seems like one would be sufficient

Reply:
They are similar but validate different parts of the rendering